ai-docs/emoji-commit-ref.yaml

```
@@ -190,3 +190,85 @@ commit_message_conventions:
+      - 'fixed'
+      - 'resolve'
+      - 'resolves'
+      - 'resolved'
+      - 'complete'
+      - 'completes'
+      - 'completed'
+      - 'closing'
+      - 'fixing'
+      - 'resolving'
+      - 'completing'
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)